### PR TITLE
Milestone 7: Add forms

### PR DESCRIPTION
**Pull request by zdnahom**
## Milestone 7: Add forms
In this milestone I :
- Added a method called 'current_user' in ApplicationController 
- Created forms for the below
  - posts(Creates a Post on behalf of the current_user (use the method that you created in your ApplicationController).)
  - comments(Create a comment on behalf of the current_user (use the method that you created in your ApplicationController))
  - likes(Allow Users to add likes to Posts.)

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -1,5 +1,17 @@
 class CommentsController < ApplicationController
-  def index; end
+  def new; end
```
Comment (emmanuellekamwa) [inline app/controllers/comments_controller.rb:5]: - [ ] The "new" method should not be blank. Use this code `@comment = Comment.new`.

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -8,4 +8,20 @@ def show
     @post = Post.find(params[:id])
     @author = User.find(params[:user_id])
   end
+
+  def new; end
+
```
Comment (emmanuellekamwa) [inline app/controllers/posts_controller.rb:15]: - [ ] The "new" method should not be blank. Use this code `@post = Post.new`.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED ⛔❌**
Hello @zdnahom 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Descriptive PR.
- Good commit messages.
- Correct git-flow.

## Required Changes ♻️
_Check the comments under the review._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (zdnahom) [reply app/controllers/comments_controller.rb:5]: I get what you mean @emmanuellekamwa, I will definitely do that 🙌

File: app/controllers/posts_controller.rb (hunk 2 above)
Comment (zdnahom) [reply app/controllers/posts_controller.rb:15]: I will add it @emmanuellekamwa 🙌

**Comment by zdnahom** [comment]
Thanks for the review @emmanuellekamwa 😉

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1,3 +1,9 @@
+<% if flash[:notice] %>
```
Comment (danmainah) [inline app/views/posts/index.html.erb:1]: - [ ] [optional] @zdnahom great work implementing the changes requested,to make the user experience better, I would suggest you add links, from the root page to take the user to the user show page and another from the user show page to posts page and from posts index page to posts show page.

**Review by danmainah — APPROVED** [review]
Hi @zdnahom , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️  Implemented changes requested, ensuring a suer can add a post comment and like.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zdnahom** [comment]
Thanks for the review @danmainah 🙏

File: app/views/posts/index.html.erb (hunk 3 above)
Comment (zdnahom) [reply app/views/posts/index.html.erb:1]: I will do that @danmainah, thanks for the suggestion 😉

**Pull request merged**